lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -108,11 +108,7 @@ def draw(self, renderer):
             [self._position3d, self._position3d + self._dir_vec], renderer.M)
         dx = proj[0][1] - proj[0][0]
         dy = proj[1][1] - proj[1][0]
-        if dx==0. and dy==0.:
-            # atan2 raises ValueError: math domain error on 0,0
-            angle = 0.
-        else:
-            angle = math.degrees(math.atan2(dy, dx))
+        angle = math.degrees(math.atan2(dy, dx))
```

Comment:
math.atan2(0, 0) (or likewise with floats) returns 0 in Py2 and Py3 and that's also mandated by the C standard...